pipelines/build/regeneration/jdk11_regeneration_pipeline.groovy

```diff
@@ -60,14 +73,20 @@ node ("master") {
     // Pull in other paramterised values (or use defaults if they're not defined)
     def jobRoot = "$JOB_ROOT" != "" ? JOB_ROOT : "build-scripts"
     def jenkinsBuildRoot = "$JENKINS_BUILD_ROOT" != "" ? JENKINS_BUILD_ROOT : "https://ci.adoptopenjdk.net/job/build-scripts/"
+    def jobTemplatePath = "$JOB_TEMPLATE_PATH" != "" ? JOB_TEMPLATE_PATH : "pipelines/build/common/create_job_from_template.groovy"
```

Comment:
`create_job_from_template.groovy` seems like a pretty standard file in that there isn't much I would change in there. So I'm failing to see the need to parameterize it. Same story for kick_off_build.

Reply:
Merely because it may as well be done incase someone else comes along and wants to replicate the adopt scripts (or in case the location of the file changes in the future). There isn't a need for it right now but there might be in the future